crypto/hd/algorithm_test.go

```diff
@@ -34,6 +34,11 @@ func init() {
 const mnemonic = "picnic rent average infant boat squirrel federal assault mercy purity very motor fossil wheel verify upset box fresh horse vivid copy predict square regret"
 
 func TestKeyring(t *testing.T) {
+	// hdWalletFixEnv defines whether the standard (correct) bip39
+	// derivation path was used, or if derivation was affected by
+	// https://github.com/btcsuite/btcutil/issues/172
+	const hdWalletFixEnv = "GO_ETHEREUM_HDWALLET_FIX_ISSUE_179"
```

Comment:
we can move this outside the test function

Reply:
Do you mean add in make file?
```test-unit-cover: export GO_ETHEREUM_HDWALLET_FIX_ISSUE_179="true" ARGS=-timeout=10m -race -coverprofile=coverage.txt -covermode=atomic```

Reply:
updated, thanks